proto/tendermint/types/types.proto

```diff
@@ -211,17 +211,17 @@ message BlobTx {
 // ShareProof is an NMT proof that a set of shares exist in a set of rows and a
 // Merkle proof that those rows exist in a Merkle tree with a given data root.
 message ShareProof {
-  repeated bytes data = 1;
-  repeated NMTProof share_proofs = 2;
-  bytes namespace_id = 3;
-  RowProof row_proof = 4;
-  uint32 namespace_version = 5;
+  repeated bytes    data              = 1;
+  repeated NMTProof share_proofs      = 2;
+  bytes             namespace_id      = 3;
+  RowProof          row_proof         = 4;
+  uint32            namespace_version = 5;
```

Comment:
called proto lint + format, not sure why our protobuf linter doesn't also call those